### DISCUSSION TITLE
How use trigger field in update action

### DISCUSSION THREAD

**Discussion by Kurt-Aziri** [Budibase/budibase#1779] [Help] [answered]
Dear,
I have a button in a table, which fires an automation (set status to 'OK').
I pass the id of the table-row (Auto ID). 
In the automation, the trigger field is known as ID
I now use this ID in an 'Update Row' action as the 'Row ID'.
It doesn't seem to work.
As an Trigger Field is always a Text type, this might be the reason why it's not working ?
How should I convert this Trigger Field to a number, in order to use it as a Row ID in the update action ?

**Comment by mike12345567** [comment, marked as answer]
Hi @Kurt-Aziri, it sounds like you might be using the `Auto ID` field as the rows ID, however this technically isn't the correct column to use when updating a row, every row as a string field of `_id` and `_rev` which are used to update a row.

I understand the confusion here, the auto ID column does suggest it is a primary key/identifying field, we simply supplied this as the `_id` UUID is not very human-readable and having an incrementing ID field is necessary for some applications.

In the future it'll be possible to lookup rows via a query in an automation, once we implement this it'll be possible to use the auto ID as the actual row ID (for updating/deleting rows as part of an automation).

**Reply by Kurt-Aziri** [reply]
Indeed. That was the problem. Using the _id did the trick. Thanks !
